nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/CopyDocument.java

```diff
@@ -45,12 +47,18 @@
     @Param(name = "name", required = false)
     protected String name;
 
+    @Param(name = CoreEventConstants.RESET_LIFECYCLE, required = false)
+    protected Boolean resetLifeCycle;
```

Comment:
it's better to use the primitif type `boolean` instead of the wrapper one and then the code can be changed to avoid the `if else` statement 
```
  CopyOption[] options = resetLifeCycle ? new CopyOption[] { CopyOption.RESET_LIFE_CYCLE } : new CopyOption[] {};
        return session.copy(doc.getRef(), target, n, options);
```